apollo-portal/src/main/resources/static/views/component/release-modal.html

```diff
@@ -51,103 +51,133 @@ <h4 class="modal-title" ng-show="toReleaseNamespace.isBranch && toReleaseNamespa
+                                <th>
+                                    {{'Component.Publish.ModifyUser' | translate }}
+                                </th>
+                                <th>
+                                    {{'Component.Publish.ModifyTime' | translate }}
+                                </th>
+                                <th>
+                                    {{'Component.Publish.Compare'|translate}}
+                                </th>
+                            </tr>
                             </thead>
                             <tbody>
-                                <tr ng-repeat="config in toReleaseNamespace.items"
-                                    ng-if="config.item.key && config.isModified">
-                                    <td width="20%" title="{{config.item.key}}">
-                                        <span ng-bind="config.item.key"></span>
-                                        <span class="label label-success" ng-if="config.isModified && !config.oldValue"
-                                            data-tooltip="tooltip" data-placement="bottom"
-                                            title="{{'Component.Publish.NewAddedTips' | translate }}">{{'Component.Publish.NewAdded' | translate }}</span>
-                                        <span class="label label-info"
-                                            ng-if="config.isModified && config.oldValue && !config.isDeleted"
-                                            data-tooltip="tooltip" data-placement="bottom"
-                                            title="{{'Component.Publish.ModifiedTips' | translate }}">{{'Component.Publish.Modified' | translate }}</span>
-                                        <span class="label label-danger" ng-if="config.isDeleted" data-tooltip="tooltip"
-                                            data-placement="bottom"
-                                            title="{{'Component.Publish.DeletedTips' | translate }}">{{'Component.Publish.Deleted' | translate }}</span>
-                                    </td>
-                                    <td width="25%" title="{{config.oldValue}}">
-                                        <span ng-bind="config.oldValue"></span>
-                                    </td>
-                                    <td width="25%" title="{{config.newValue}}">
-                                        <span ng-bind="config.newValue"></span>
-                                    </td>
-                                    <td width="15%" ng-bind="config.item.dataChangeLastModifiedBy">
-                                    </td>
-                                    <td width="15%"
-                                        ng-bind="config.item.dataChangeLastModifiedTime | date: 'yyyy-MM-dd HH:mm:ss'">
-                                    </td>
-                                </tr>
+                            <tr ng-repeat="config in toReleaseNamespace.items"
```

Comment:
please keep the origin format to avoid unnecessary changes

Reply:
ok,I will do it.Thank you